elb_automation/meaoelb/config.py

```diff
@@ -0,0 +1,153 @@
+import pprint
+
+
+class DictLike(dict):
```

Comment:
FYI Python 3.7 data classes look like they might be good for this use case in the future: https://www.python.org/dev/peps/pep-0557/